common-test-lib/src/main/java/com/google/gct/idea/testing/TestUtils.java

```diff
@@ -119,6 +120,16 @@ private static Disposable getParentDisposableForCleanup() {
   }
 
   @NotNull
+  public static <T> T installMockService(@NotNull Class<T> serviceInterface) {
```

Comment:
Ignore the last comment - I must have checked out the branch in an earlier state - all tests pass.